pandas/tests/frame/methods/test_join.py

```diff
@@ -104,6 +107,145 @@ def test_suffix_on_list_join():
     tm.assert_frame_equal(arr_joined, norm_joined)
 
 
+@pytest.fixture
```

Comment:
can you put these at the top of the file (the fixtures)

Reply:
done!